app/main/views/services.py

```diff
@@ -51,14 +51,28 @@ def view(service_id):
         flash({'api_error': service_id}, 'error')
         return redirect(url_for('.index'))
 
+    last_updated_by = last_updated_date = None
+    if service_data['status'] != 'published':
+        most_recent_audit_events = data_api_client.find_audit_events(
+            latest_first="true",
+            object_id=service_id,
+            object_type="services",
+            audit_type=AuditTypes.update_service_status
+        )
+        if most_recent_audit_events.get('auditEvents'):
+            last_updated_by = most_recent_audit_events['auditEvents'][0]['user']
```

Comment:
I still like `removed_by` and `removed_on` for these variables...

Reply:
Maybe `removed_at` if it's a full timestamp, actually.

Reply:
Whoops - I thought I'd changed that. Now fixed.